### ISSUE TITLE
GET http://localhost:8080/images/icons/Generic.png 404 (Not Found)

### ISSUE THREAD

**Issue by wzhsunn** [librenms/librenms#576]
I think there is some resource missing .  
Chrome developer tools show this error :

```
GET http://localhost:8080/images/icons/Generic.png 404 (Not Found)    (index):88
```

**Comment by f0o** [comment]
On what page does that trigger?

**Comment by wzhsunn** [comment]
Menubar --> Deivcies --> All Devices --> Generic 

The image of  Generic is broken .

**Comment by f0o** [comment]
is your repo updated?

Might have an idea where it relies..

**Comment by f0o** [comment]
Ok the bug is known and going to be resolved in near future, it's just a typo and a missing picture for non-categorized devices.

**Comment by laf** [comment]
This is being fixed in #577
